core/unit_test/TestAtomicOperations_double.hpp

```diff
@@ -21,10 +21,13 @@ TEST(TEST_CATEGORY, atomic_operations_double) {
   const int start = 1;  // Avoid zero for division.
   const int end   = 11;
   for (int i = start; i < end; ++i) {
+#ifndef KOKKOS_ENABLE_OPENACC
+    // FIXME_OPENACC: OpenACC C/C++ does not support atomic min/max operations
```

Comment:
Did you consider guarding in `AtomicOperationsTestNonIntegralType` instead?

Reply:
Changed the way to guard unsupported atomic tests for the OpenACC backend.